dataapiv1/spec.yaml

```diff
@@ -61,10 +62,43 @@ paths:
             X-CB-Flags:
               $ref: "#/components/headers/DocumentFlags"
           content:
+            'application/json':
```

Comment:
This change might be better described specifically as the 3 cases we support:
https://github.com/couchbase/stellar-gateway/blob/master/gateway/dapiimpl/server_v1/helpers.go#L74-L85

Reply:
Done!